### PR TITLE
Add a scaffold operations worker for ansible tower

### PR DESCRIPTION
**Pull request by agrare**
Adds a simple worker that waits for operations commands on the operations kafka topic.

### CODE REVIEW

File: lib/topological_inventory/ansible_tower/operations/worker.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          end
+        ensure
+          client&.close
+        end
+
+        private
+
+        attr_accessor :messaging_client_opts
+
+        def process_message(message)
```
Comment (slemrmartin) [inline lib/topological_inventory/ansible_tower/operations/worker.rb:31]: it can call `Processor.process` in this PR, it's generic code too

**Review by slemrmartin — APPROVED** [review]
:+1: Nice backbone, it would be great to extract this code to common (ingress-api-client?), most of the code can be shared with openshift-operations (+will be shared with azure?)

**Comment by agrare** [comment]
> it would be great to extract this code to common (ingress-api-client?)

Eh I would prefer not to put any operations code in the ingress-api-client.  I'm not a huge fan of some of the collector common code there in the first place and this has even less to do with ingress-api.  If anything collector-common and operations-common gems would be better.

File: lib/topological_inventory/ansible_tower/operations/worker.rb (hunk 1 above)
Comment (agrare) [reply lib/topological_inventory/ansible_tower/operations/worker.rb:31]: :+1: added with specs

**Comment by Fryguy** [comment]
Nice.  +1 for the shared code idea, but not in a client...we need some sort of shared topo gem...perhaps topo core?  Either way, that's for the future.

**Comment by agrare** [comment]
Yeah actually topo-core could be a good place :+1:

**Pull request merged**